requirements/edx/base.in

```diff
@@ -178,3 +178,4 @@ xss-utils                           # https://github.com/edx/edx-platform/pull/2
 enmerkar-underscore                 # Implements a underscore extractor for django-babel.
 xblock-drag-and-drop-v2             # Drag and Drop XBlock
 xblock-google-drive                 # XBlock for google docs and calendar
+openedx-django-require
```

Comment:
Please alphabetize this in the list above.

Reply:
done